js/src/modals/AddAddress/addAddress.js

```diff
@@ -28,6 +28,7 @@ export default class AddAddress extends Component {
 
   static propTypes = {
     contacts: PropTypes.object.isRequired,
+    address: PropTypes.string,
```

Comment:
General comment. When changing these (or adding, removing), it would make sense to start putting them in alphabetical order since you are in there already. (It goes for everywhere - low overhead). This one should have been on-top as an example.